modules/primer-marketing-utilities/lib/layout.scss

```diff
@@ -15,3 +15,35 @@
     .position-#{$breakpoint}-fixed    { position: fixed !important; }
   }
 }
+
+// Negative offset columns
+.offset-0 { margin-left: 0; }
+.offset-n1 { margin-left: -(1 / 12 * 100%); }
+.offset-n2 { margin-left: -(2 / 12 * 100%); }
+.offset-n3 { margin-left: -(3 / 12 * 100%); }
+.offset-n4 { margin-left: -(4 / 12 * 100%); }
+.offset-n5 { margin-left: -(5 / 12 * 100%); }
+.offset-n6 { margin-left: -(6 / 12 * 100%); }
+.offset-n7 { margin-left: -(7 / 12 * 100%); }
+.offset-n8 { margin-left: -(8 / 12 * 100%); }
+.offset-n9 { margin-left: -(9 / 12 * 100%); }
+.offset-n10 { margin-left: -(10 / 12 * 100%); }
+.offset-n11 { margin-left: -(11 / 12 * 100%); }
+
+// Responsive classes for negative offset columns
+@each $breakpoint in map-keys($breakpoints) {
```

Comment:
These are gonna add a ton of new CSS. Do you think we need all of the offsets? Or can we get away with just the first three or so?

Anything above `offset-n3` is gonna be outside of the viewport on most screens (specifically, assuming you have a div negatively offset by 3 within `container-xl`, it will be off-canvas on all screens <1706px wide). As long as we're not trying to offset very large divs, I'm not sure why we'd need these values up past about 3 or 4. Here's a bad sketch that demonstrates what I'm trying to explain with words:

![image](https://user-images.githubusercontent.com/527589/50866021-f425ed00-136d-11e9-9a0f-994db8a5ffb6.png)